src/frame.rs

```diff
@@ -869,7 +874,7 @@ impl Frame {
                 stream_id: *stream_id,
                 offset: *offset,
                 length: *length as u64,
-                fin: *fin,
+                fin: if *fin { Some(true) } else { None },
```

Comment:
This is probably the most valuable change, saving some extra bytes for nearly every stream event that is logged